app/src/main/res/layout/fragment_upload_list.xml

```diff
@@ -8,23 +9,38 @@
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/rv_upload_list"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="match_parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
 
     </androidx.recyclerview.widget.RecyclerView>
 
     <TextView
         android:id="@+id/tv_no_uploads"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
         android:layout_centerInParent="true"
+        android:layout_marginEnd="411dp"
+        android:layout_marginBottom="731dp"
```

Comment:
This won't look good for all screen sizes. Please make this Textview at the center.

Reply:
Thanks for the feedback. I have corrected that and pushed the changes.